nemo/collections/nlp/models/language_modeling/megatron/bert_model.py

```diff
@@ -155,15 +166,18 @@ def __init__(
         hidden_dropout=0.1,
         precision=16,
         fp32_residual_connection=False,
+        activations_checkpoint_granularity=None,
         activations_checkpoint_method=None,
         activations_checkpoint_num_layers=1,
+        activations_checkpoint_layers_per_pipeline=None,
```

Comment:
Shouldn't num_micro_batches_with_partial_activation_checkpoints also be set somewhere here? Not sure

Reply:
Following what ever was on GPT3.